### ISSUE TITLE
[DatePicker] Accessibility (navigation mode: default) - Selected date cell of MaterialCalendar is not announcing as a selected date.

### ISSUE THREAD

**Issue by KarmaBuildsApps** [material-components/material-components-android#1154]
**Last tested version** 
-  material:1.2.0-alpha05

**Description:** 
When doing talkback swiping on calendar date cells and if the current swipe focus is on current selected date cell, talkback doesn't announce the cell date as a selected. If one were visually blind he/she wouldn't know what is currently selected (especially if one want to confirm his/her selection before swiping to ok button and clicking on it.)  

**Expected behavior:** 
Selected date cell should announce as a selected date.

**Android API version:** 
Tested on Android API version 28

**Material Library version:** Material Android Library version you are using material:1.2.0-alpha05

**Device:** Device on which the bug was encountered
- Oneplus 5
